src/test/java/ca/corefacility/bioinformatics/irida/service/export/ExportUploadServiceTest.java

```diff
@@ -41,7 +41,7 @@
 
 	@Test
 	public void testUploadSubmission() throws UploadException, IOException {
-		NcbiExportSubmission submission = createFakeSubmission(".fastq");
+		NcbiExportSubmission submission = createFakeSubmission();
```

Comment:
This file too (https://github.com/phac-nml/irida/pull/564)

Reply:
Fixed thanks